containers/centos/stream10/Containerfile

```diff
@@ -0,0 +1,10 @@
+#
+# A CentOS Stream 10 image tailored for tmt test suite
+#
+# tmt/tests/centos/stream10/upstream:latest
+#
+
+FROM quay.io/centos/centos:stream10
+
+    # Populate dnf cache
```

Comment:
Weird indent

Reply:
Expected, aligns with the command, not `RUN`.

Reply:
ok, we have different state, but I can live with it :)